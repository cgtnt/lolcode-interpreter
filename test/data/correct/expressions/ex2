HAI

VISIBLE ALL OF WIN AN WIN AN WIN MKAY!
VISIBLE ALL OF WIN AN FAIL MKAY!
VISIBLE ANY OF FAIL AN FAIL AN WIN MKAY!
VISIBLE ANY OF FAIL AN FAIL MKAY!

VISIBLE BOTH OF WIN AN WIN!
VISIBLE BOTH OF WIN AN FAIL!
VISIBLE EITHER OF WIN AN FAIL!
VISIBLE EITHER OF WIN AN WIN!
VISIBLE WON OF WIN AN WIN!
VISIBLE NOT FAIL!

KTHXBYE
