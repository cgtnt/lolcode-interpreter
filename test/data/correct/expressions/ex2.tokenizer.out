@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_WRITE_STDOUT:VISIBLE-@3:T_BOOL_AND_INF:ALL OF-@3:T_TRUE:WIN-@3:T_AND:AN-@3:T_TRUE:WIN-@3:T_AND:AN-@3:T_TRUE:WIN-@3:T_END_INF:MKAY-@3:T_BANG:!-@4:T_COMMAND_TERMINATOR:
-@4:T_WRITE_STDOUT:VISIBLE-@4:T_BOOL_AND_INF:ALL OF-@4:T_TRUE:WIN-@4:T_AND:AN-@4:T_FALSE:FAIL-@4:T_END_INF:MKAY-@4:T_BANG:!-@5:T_COMMAND_TERMINATOR:
-@5:T_WRITE_STDOUT:VISIBLE-@5:T_BOOL_OR_INF:ANY OF-@5:T_FALSE:FAIL-@5:T_AND:AN-@5:T_FALSE:FAIL-@5:T_AND:AN-@5:T_TRUE:WIN-@5:T_END_INF:MKAY-@5:T_BANG:!-@6:T_COMMAND_TERMINATOR:
-@6:T_WRITE_STDOUT:VISIBLE-@6:T_BOOL_OR_INF:ANY OF-@6:T_FALSE:FAIL-@6:T_AND:AN-@6:T_FALSE:FAIL-@6:T_END_INF:MKAY-@6:T_BANG:!-@8:T_COMMAND_TERMINATOR:
-@8:T_WRITE_STDOUT:VISIBLE-@8:T_BOOL_AND:BOTH OF-@8:T_TRUE:WIN-@8:T_AND:AN-@8:T_TRUE:WIN-@8:T_BANG:!-@9:T_COMMAND_TERMINATOR:
-@9:T_WRITE_STDOUT:VISIBLE-@9:T_BOOL_AND:BOTH OF-@9:T_TRUE:WIN-@9:T_AND:AN-@9:T_FALSE:FAIL-@9:T_BANG:!-@10:T_COMMAND_TERMINATOR:
-@10:T_WRITE_STDOUT:VISIBLE-@10:T_BOOL_OR:EITHER OF-@10:T_TRUE:WIN-@10:T_AND:AN-@10:T_FALSE:FAIL-@10:T_BANG:!-@11:T_COMMAND_TERMINATOR:
-@11:T_WRITE_STDOUT:VISIBLE-@11:T_BOOL_OR:EITHER OF-@11:T_TRUE:WIN-@11:T_AND:AN-@11:T_TRUE:WIN-@11:T_BANG:!-@12:T_COMMAND_TERMINATOR:
-@12:T_WRITE_STDOUT:VISIBLE-@12:T_BOOL_XOR:WON OF-@12:T_TRUE:WIN-@12:T_AND:AN-@12:T_TRUE:WIN-@12:T_BANG:!-@13:T_COMMAND_TERMINATOR:
-@13:T_WRITE_STDOUT:VISIBLE-@13:T_BOOL_NOT:NOT-@13:T_FALSE:FAIL-@13:T_BANG:!-@15:T_COMMAND_TERMINATOR:
-@15:T_END:KTHXBYE-@16:T_COMMAND_TERMINATOR:
-@16:T_EOF:
